src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -60,9 +61,12 @@ public HelpWindow() {
         helpBox.initModality(Modality.NONE);
         helpBox.setResizable(true);
         helpBox.getDialogPane().setPrefWidth(900.0);
-        helpBox.getDialogPane().setPrefHeight(530.0);
+        helpBox.getDialogPane().setPrefHeight(450.0);
         helpBox.getDialogPane().getStylesheets().add("/view/DarkTheme.css");
         helpBox.getDialogPane().getStyleClass().add("helpBox");
+        helpBox.getDialogPane().setStyle("-fx-font-family: 'Courier New';");
+        helpBox.setX(200);
+        helpBox.setY(125);
```

Comment:
yeah, i put it back cause its not working, but after putting it back it still doesnt work, but its okay at least it works on your computer cause all i need is a screenshot to add to the UG. And yes its from current master